abstract_notebook.ipynb

```diff
@@ -39,7 +39,7 @@
     {
```

Comment:
<p>Why one more line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xuhdev/notebook-maniac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Yes I think we need two more lines</p>